src/client.js

```
@@ -21,13 +22,28 @@ const viewBuildTypes = [
 const client = mozaik => {
 
     mozaik.loadApiConfig(config);
+    const caFilePath = config.get('jenkins.customCa');
+    let certificate;
+
+    if (caFilePath.length > 0) {
+        try {
+            certificate = fs.readFileSync(caFilePath);
+        } catch(error) {
+            mozaik.logger.error(chalk.red(`[jenkins] an error occurred while trying to read custom certificate (${ error })`));
```

you should throw the error